src/Codeception/Module/Facebook.php

```diff
@@ -6,6 +6,7 @@
 use Codeception\Lib\Driver\Facebook as FacebookDriver;
 use Codeception\Lib\Interfaces\DependsOnModule;
 use Codeception\Lib\Interfaces\RequiresPackage;
+use Codeception\Module\Notification;
```

Comment:
It is `Codeception\Lib\Notification` - https://github.com/Codeception/Codeception/blob/2.3/src/Codeception/Lib/Notification.php